com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -202,8 +202,24 @@ public void testTf2()
         0); // NOTE: Change to testTf2("tf2_test_dataset.py", "add", 2, 3, 2, 3) once
     // https://github.com/wala/ML/issues/89 is fixed.
     testTf2("tf2_test_tensor_list.py", "add", 2, 3, 2, 3);
-    testTf2("tf2_test_tensor_list2.py", "add", 0, 2);
-    testTf2("tf2_test_tensor_list3.py", "add", 0, 2);
+    //    testTf2("tf2_test_tensor_list2.py", "add", 0, 2);
+    //    testTf2("tf2_test_tensor_list3.py", "add", 0, 2);
+    testTf2(
+        "tf2_test_model_call.py",
+        "SequentialModel.__call__",
+        0,
+        3); // NOTE: Change to testTf2("tf2_test_model_call.py", "SequentialModel.__call__", 1, 5,
+    // 2) once
+    // https://github.com/wala/ML/issues/24 is fixed.
+    testTf2(
+        "tf2_test_model_call2.py",
+        "SequentialModel.call",
+        0,
+        3); // NOTE: Change to testTf2("tf2_test_model_call.py", "SequentialModel.call", 1, 5, 2)
+    // once
```

Comment:
`tf2_test_model_call.py` is already in use by the previous test.

Reply:
I don't understand your comment. Can you elaborate?